natalie.html

```diff
@@ -0,0 +1,182 @@
+      
+  </div>
+
+  <!--Playlist to listen to whilst coding: [https://open.spotify.com/playlist/597avdsU0EaVNEMrvKB2Cc?si=1ri-R_x9R8uiF3E9PC5QNg] wanna play while visiting the website-->
+
+  <section id="hobbies">
+
+   <h2> Hobbies & Interests </h2> <!-- again a more interactive design with no so much text, thinking on incorporating audio files-->
+   <p> I do yoga most days, and when the gyms are open, I enjoy weightlifting at Black Country Barbell. I'm currently punishing myself by playing a lot of Warzone.
+       I enjoy collecting vinyls and playing piano, when I have the time! I also like to learn crafts such as sewing and knitting, but also digital making using my Raspberry Pi.
+       I'm an Aston Villa fan, and also follow F1 and NFL.
+     </p>
+
+
+   <div class="row">
+    <div class="column">
+      <img src="yoga.jfif">
+      <img src="sub18weight-articleLarge.jpg">
+      <img src="gym.jpg">
+      <img src="warzone_cover_art.jpg">
+    </div>
+
+    <div class="column">
+      <img src="piano.jpg">
+      <img src="vinyl.jpg">
+      <img src="sewing.jpg">
+      <img src="raspberry.jfif">
+      
+     
+    </div>
+
+    <div class="column">
+      <img src="nfl.jpg">
+      <img src="villa.jpg">
+      <img src="escudo.jpg">
+      <img src="formula.jpeg">
+          
+    </div>
+
+    
+   </div>
+  
+
+   
+</section>
+
+<section id="guess">
+
+    <h2>  I dare you to guess! Two truths and a lie <!--wink emoji, need to fix the the boolean that checks if you are right or wrong--> </h2>
+    <!-- maybe also incorporate sounds when chosing the right/wrong answer, maybe a pic of her doing so lol-->
+    <div class="container">
+      <ul>
+        <li><label>
+        <input type="checkbox" name="option"> Some of my family is from the Bahamas  </label></li> 
+        <li><label>
+        <input type="checkbox" name="option">  I have run the London Marathon </label> </li> 
+        <li><label>
+        <input type="checkbox" name="option"> Even though I teach Maths I failed my Maths A-Level </label> </li> 
+      </ul>
+    </div>
+
+
+ </section>
+
+ <section  id="contact">
+
+   <h2> Get in touch here </h2>
+   
+     <form>
+  
+       <label for="name"> Your Name</label>
+       <br>
+       <input type="text" id="fname" name="fullname" placeholder="Your name..">
+       <br>
+ 
+       <label for="email"> Email</label>
+       <br>
+       <input type="text" id="femail" name="email" placeholder="Your email.."> 
+       <br>
+ 
+       <label for="text"> Phone number</label> 
+       <br>
+       <input type="text" id="fphone" name="phone" placeholder="Your telephone.."> 
+       <br> 
+       
+   
+       <label for="subject">Subject</label>
+       <br>
+       <textarea id="subject" name="subject" placeholder="Write something.." style="height:200px"></textarea>
+       <br>
+   
+       <input type="submit" value="Submit">
+   
+    </form>
+  </section>
+
+</div>
+   
+  
+
+
+<div class="footer">

```

Comment:
Semantic HTML has tags for entire structure. We can replace div with class footer for the footer tag. 

```suggestion
<footer>

```